test/scale-infra/deploy_ks_core.yaml

```diff
@@ -0,0 +1,130 @@
+             kubectl delete crds $(kubectl get crds | grep kubestellar) --ignore-not-found
+             kubectl --context kscore delete -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/refs/tags/controller-v1.12.0/deploy/static/provider/kind/deploy.yaml  --ignore-not-found
+      when: result is success
+
+
+    - name: install ingress controller
+      shell: |
+             kubectl config rename-context kubernetes-admin@kubernetes kscore
+             kubectl --context kscore label node --all ingress-ready="true"
+             kubectl --context kscore apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/refs/tags/controller-v1.12.0/deploy/static/provider/kind/deploy.yaml
+
+    - name: patching nginx ingress to enable SSL passthrough
+      shell: |
+             kubectl --context kscore patch deployment ingress-nginx-controller -n ingress-nginx -p '{"spec":{"template":{"spec":{"containers":[{"name":"controller","args":["/nginx-ingress-controller","--election-id=ingress-nginx-leader","--controller-class=k8s.io/ingress-nginx","--ingress-class=nginx","--configmap=$(POD_NAMESPACE)/ingress-nginx-controller","--validating-webhook=:8443","--validating-webhook-certificate=/usr/local/certificates/cert","--validating-webhook-key=/usr/local/certificates/key","--watch-ingress-without-class=true","--publish-status-address=localhost","--enable-ssl-passthrough"]}]}}}}'
+
+    - name: extract the port number for the ingress controller
+      shell: kubectl --context kscore -n ingress-nginx get svc ingress-nginx-controller -o=jsonpath='{.spec.ports[1].nodePort}'
+      register: ingress_port
+
+    - name: install KubeStellar using core helm-cart
+      shell: |
+             export KUBESTELLAR_VERSION="{{ ks_release }}"
+             helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
+             --version $KUBESTELLAR_VERSION \
+             --set-json='ITSes=[{"name":"its1","type":"host"}]' \
+             --set-json='WDSes=[{"name":"wds1"}]' \
+             --set-json='verbosity.default=5' \
+             --set kubeflex-operator.externalPort={{ ingress_port.stdout }}
+
+    - name: Get kubeconfig "context" for wds1 space
+      shell: |
+             kflex ctx wds1
+
+    - name: Get the token to register managed clusters
+      shell: clusteradm --context kscore get token | grep '^clusteradm join'
+      register: wec_join_command
+      until: "wec_join_command is not failed"
+      retries: 5
+      delay: 15
+             
+    -  name: write wec_join_command to file
+       ansible.builtin.copy:
+         content: "{{ wec_join_command.stdout }}"
+         dest:  /tmp/{{ region }}_{{ vpc_name }}_wec_join_command
+    
+    - name: Copy join command to local file (Ansible control node)
+      ansible.builtin.fetch:
+        src: /tmp/{{ region }}_{{ vpc_name }}_wec_join_command
+        dest: .data/
+        flat: yes
+
+    - name: Set some environment configurations for the end user
```

Comment:
"some environment configurations" is pretty vague. I would say something like "Set the current kubeconfig context to the KubeFlex hosting cluster"

Reply:
Fixed!